### PR TITLE
G Suite: Update Thank You page and nudge for Google Workspace

### PR DESCRIPTION
**Pull request by hambai**
#### Changes proposed in this Pull Request

* update the checkout thank you page to use the proper product name
* update the checkout gsuite nudge to use the active google mail service name

#### Testing instructions

* You can open an existing G Suite thank you page with `/checkout/thank-you/{domain}/{receiptID}` to verify how that looks
* I'm not sure how to test the nudge as I don't think it's referenced anywhere

### CODE REVIEW

**Review by sirbrillig — APPROVED** [review]
This looks good to me. You should probably get a review from @stephanethomas as well just to be sure, but I don't see any issues. IIRC you're correct that the GSuiteNudge is not active anywhere right now but you _can_ force it by visiting /checkout/:site/with-gsuite/:domain/:receiptId? (eg: http://calypso.localhost:3000/checkout/example.com/with-gsuite/example.com)

File: client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx
Comment (stephanethomas) [inline client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx:68]: It may be a good idea to add a comment here and in other translations to explain what is `productName`.

File: client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx
Comment (stephanethomas) [inline client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx]: In this particular case, I feel using the family name instead of the product name would be more appropriate.

File: client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx
Comment (stephanethomas) [inline client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx]: I would use the product family name here as well.

File: client/my-sites/checkout/checkout-thank-you/header.jsx
Comment (stephanethomas) [inline client/my-sites/checkout/checkout-thank-you/header.jsx]: Minor but the product name is incorrect:
```suggestion
						comment: '%(productName)s can be either "G Suite" or "Google Workspace Business Starter"',
```

**Review by stephanethomas — APPROVED** [review]
Some tests are still failing, which we must fix. Other than that, I only had suggestions about using the product family name.

Feel free to ship it once this has been addressed 🚢.

**Comment by stephanethomas** [comment]
Still looks good to me. Please ship it 👍.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5472044

Hi @hambai, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Learn more about %(productFamily)s`
- `Add %(googleMailService)s < %(siteTitle)s`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5472044).

**Pull request merged**
